### ISSUE TITLE
bug: okta source plugin unable to unmarshal okta_applications tables 

### ISSUE THREAD

**Issue by von-cloudops** [cloudquery/cloudquery#16832]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

okta_application tables are not getting synced. I am seeing the following errors. 

truncated plugin log line line="{\"level\":\"error\",\"module\":\"okta-src\",\"table\":\"okta_applications\",\"client\":\"okta\",\"error\":\"received error from Okta API: err: Failed to unmarshal ListApplications200ResponseInner as OpenIdConnectApplication: json: cannot unmarshal bool into Go struct field _Application.profile of type map[string]interface {}, body:

### Expected Behavior

it should sync data into okta_application_users & okta_application_group_assignments tables

### CloudQuery (redacted) config

```kind: source
spec:
  # Source spec section
  name: okta
  path: cloudquery/okta
  registry: cloudquery
  version: "v4.0.1"
  tables: ["*"]
  destinations: ["postgresql"]
  spec:
    # Required. Your Okta domain name
    domain: <<Domain Name>>

    # Optional. Okta Token to access API
    # ⚠️ Warning - Your token should be kept secret and not committed to source control.
    # ⚠️ Warning - In the future versions token parameter will become required.
    token: << Token >>

    # Optional. Rate limiter settings
    concurrency: 200
    rate_limit:
      max_backoff: 65s
      max_retries: 3
```

### Steps To Reproduce

use this configuration and run sync. 

### CloudQuery (redacted) logs

truncated plugin log line line="{\"level\":\"error\",\"module\":\"okta-src\",\"table\":\"okta_applications\",\"client\":\"okta\",\"error\":\"received error from Okta API: err: Failed to unmarshal ListApplications200ResponseInner as OpenIdConnectApplication: json: cannot unmarshal bool into Go struct field _Application.profile of type map[string]interface {}, body:

### CloudQuery version

3.29.1

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by erezrokah** [comment]
I'll re-open so we can take a look

**Comment by erezrokah** [comment]
Worth trying to update to https://github.com/cloudquery/cloudquery/pull/16831

**Comment by erezrokah** [comment]
Closing in favor of https://github.com/cloudquery/cloudquery/issues/16836

**Comment by disq** [comment]
@von-cloudops Hi, thanks for reporting. I'm having trouble reproducing this, can you please tell me which Okta application (or is it a custom one?) you are having this issue with?

In the meantime we've released [Okta source 4.1.0](https://hub.cloudquery.io/plugins/source/cloudquery/okta) which updates the Okta SDK, can you give it a try?

**Comment by disq** [comment]
Closing as stale, @von-cloudops if you're still hitting this issue please comment and I'll re-open
